Data/HashMap/Base.hs

```diff
@@ -623,6 +624,17 @@ lookupCont absent present !h0 !k0 !m0 = go h0 k0 0 m0
         | otherwise = absent (# #)
 {-# INLINE lookupCont #-}
 
+-- | /O(log n)/ Return the value to which the specified key is mapped,
+-- or 'Nothing' if this map contains no mapping for the key.
+--
+-- This is a flipped version of 'lookup'.
+--
+-- @since 0.2.9
```

Comment:
We sure about that number?

Reply:
Out of date since last update, was just waiting for the findWithDefault fix to go into master since the CI was failing on this PR too. Have updated to 0.2.11 and updated CHANGES.md appropriately as well. Most recent commit should have the updated annotation.